src/moonstone-samples/lib/LabeledTextItemSample.js

```diff
@@ -12,7 +12,7 @@ module.exports = kind({
 		{kind: Scroller, classes: 'enyo-fill', components: [
 			{
 				kind: LabeledTextItem,
-				label: 'Breaking Bad',
+				label: 'Breaking Bad Or Some Reeeeeeeally Loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong Title',
```

Comment:
Repeated characters aren't the dest at demonstrating marqueeing, since on a low framerate you can't tell which direction or if they're moving. I'd be in favor of dropping this line's change.